css-grid-classes/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <h1>Choose your character!</h1>
```

Comment:
Content in a `row` element should always be in a `column` element of some sort. This will take the full width, so you should use a column measurement that will take the full width of the available area